app/Repositories/LikeTweetRep.php

```diff
@@ -29,4 +29,9 @@ public function liked(LikeTweetEntity $likeTweetEntity)
     {
         return $this->likeTweet->where('user_id', $likeTweetEntity->getUser()->getId())->where('tweet_id', $likeTweetEntity->getTweet()->getId())->exists();
     }
+
+    public function countLikes($tweetId)
+    {
+        return $this->likeTweet->where('tweet_id', $tweetId)->get()->count();
```

Comment:
The problem here is do `->get()` first. this will load the whole likes object and then does array count.

instead you can simply use count only
```php
return $this->likeTweet->where('tweet_id', $tweetId)->count();
```

This way you just get the count number from MySQL directly